handlers/session_sync_handler.go

```diff
@@ -25,19 +24,19 @@ func (h *Handler) SessionSyncHandler(w http.ResponseWriter, req *http.Request, p
 	for _, adapter := range adapters {
 		meshAdapters, _ = h.addAdapter(req.Context(), meshAdapters, prefObj, adapter.Location, provider)
 	}
-	logrus.Debugf("final list of active adapters: %+v", meshAdapters)
+	h.log.Debug("final list of active adapters: ", meshAdapters)
 	prefObj.MeshAdapters = meshAdapters
 	err := provider.RecordPreferences(req, user.UserID, prefObj)
 	if err != nil { // ignoring errors in this context
-		logrus.Errorf("unable to save session: %v", err)
+		h.log.Error(ErrSaveSession(err))
 	}
 
 	if prefObj.K8SConfig != nil && h.kubeclient != nil {
 		if prefObj.K8SConfig.ServerVersion == "" {
 			// fetching server version, if it has not already been
 			version, err := h.kubeclient.KubeClient.ServerVersion()
 			if err != nil {
-				logrus.Errorf("unable to ping the Kubernetes server")
+				h.log.Error(ErrFetchKubernetes)
```

Comment:
Hey @Chadha93 Is this variable declared in the `error.go` as I could not find it.🤔

Reply:
![Screenshot from 2021-07-17 23-47-09](https://user-images.githubusercontent.com/65453826/126046208-7c67ee52-33a6-4e67-9621-62f398846132.png)
Yepp Its there

Reply:
Okay so the `ErrFetchKubernetes` is written as a function but here it was passed as a variable so the lint check failed so pass the error parameter also here. This will solve the problem I guess.